website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "Metrics"
 id: "metrics"
+description: "When you define metrics in dbt projects, you can encode crucial business logic in tested, version-controlled code."
```

Comment:
I added this so this description shows up in Google searches. Feel free to edit!